### PR TITLE
[1LP][RFR] Fixes Ansible crud test.

### PR DESCRIPTION
**Pull request by sbulage**
PRT run:
{{ pytest: cfme/tests/ansible/test_embedded_ansible_crud.py -qsvvv }}

### CODE REVIEW

**Comment by valaparthvi** [comment]
Changes LGTM :+1:

File: cfme/tests/ansible/test_embedded_ansible_crud.py [hunk 1]
```diff
@@ -35,12 +29,8 @@ def test_embedded_ansible_enable(enabled_embedded_appliance):
...
     assert wait_for(func=lambda: enabled_embedded_appliance.is_rabbitmq_running, num_sec=30)
     assert wait_for(func=lambda: enabled_embedded_appliance.is_nginx_running, num_sec=30)
-    if not enabled_embedded_appliance.is_pod:
-        endpoint = 'ansibleapi'
-        imprint = 'Ansible Tower'
-    else:
-        endpoint = 'api'
-        imprint = 'AWX'
+    endpoint = "api" if enabled_embedded_appliance.is_pod else "ansibleapi"
+    imprint = 'AWX'
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_crud.py]: @sbulage 
I think ```imprint``` is used once here. 
So we don't need this extra variable; we can directly write ```p='AWX'```.

Comment (digitronik) [inline cfme/tests/ansible/test_embedded_ansible_crud.py]: Q:
why you put this in `variable` as it used one time only in test?

Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_crud.py]: I will remove it.

Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_crud.py]: Removed it.

**Pull request merged**